basic_module/account/src/impls.rs

```diff
@@ -39,3 +39,53 @@ impl CheckTxHandler for Handler {
         }
     }
 }
+
+#[allow(dead_code)]
+pub struct Executor {}
+
+impl TransactionExecutor for Executor {
+    fn execute_transactions(
+        &self,
+        _context: &mut dyn SubStorageAccess,
+        transactions: &[SignedTransaction],
+    ) -> Result<Vec<TransactionExecutionOutcome>, ErroneousTransactions> {
```

Comment:
I modified the return type of `execute_transactions` to return Result<Vec<TransactionExecutionOutcome>, ()>. Let's implement the module fit into this type before a suitable host-side handling for ErroneousTransactions is equipped.

Reply:
It means that we don't need `ErroneousTransactions`? If it returns `Result<Vec, ()>`, what should the module return?

Reply:
We may need it in the future after host provides a suitable handling and a module decided to get help from the functionality.